extension/src/openvic-extension/utility/Utilities.hpp

```diff
@@ -92,6 +92,28 @@ namespace OpenVic::Utilities {
 		godot::Image::Format format = godot::Image::Format::FORMAT_RGBA8
 	);
 
+	static thread_local memory::vector<godot::Array> _formatting_array_pool;
```

Comment:
thread_local is by default static, this aside `_formatting_array_pool` should be hidden